InventoryBuilder.py

```diff
@@ -1,25 +1,17 @@
-import urllib3.exceptions
-from flask import Flask
-from gevent.pywsgi import WSGIServer
-from threading import Thread
 from tools.Vrops import Vrops
 from tools.helper import yaml_read
 from collections import defaultdict
+from tools.InventoryApi import InventoryApi
```

Comment:
I'm not sure if `InventoryApi` is a tool. I would prefer to group both inventory parts in one folder. e.g. `/inventory`

Reply:
Valid point, it deserves its own directory now.